lib/yabeda/sidekiq/server_middleware.rb

```diff
@@ -20,8 +21,17 @@ def call(worker, job, queue)
             yield
           end
           Yabeda.sidekiq_jobs_success_total.increment(labels)
-        rescue Exception # rubocop: disable Lint/RescueException
-          Yabeda.sidekiq_jobs_failed_total.increment(labels)
+        rescue Exception => e # rubocop: disable Lint/RescueException
+          sidekiq_jobs_failed_labels = labels.dup
+          if (config_value = Yabeda::Sidekiq.config.label_for_error_class_on_sidekiq_jobs_failed)
+            label = if config_value.is_a?(Symbol) || config_value.is_a?(String)
+                      config_value
+                    else
+                      DEFAULT_JOB_FAILED_ERROR_LABEL_KEY
+                    end
+            sidekiq_jobs_failed_labels[label] = e.class.name
+          end
```

Comment:
Why is that flexibility needed? Wouldn't just `error` tag be enough?
```suggestion
          if Yabeda::Sidekiq.config.label_for_error_class_on_sidekiq_jobs_failed
            sidekiq_jobs_failed_labels[:error] = e.class.name
          end
```